qa/tasks/ceph_salt.py

```diff
@@ -61,15 +61,44 @@ class CephSalt(Task):
                         using zypper.
         branch:         Ceph-salt branch in case repo is provided. If no branch
                         is provided master is used by default.
+        containers:
+            registry:
+                name: registry name
+                location: <registry url/IP:port>
+                insecure: True/False    (Is registry insecure, True by default)
+            image:  <container image path>
 
+
+    For example:
+
+    containers:
+        registry:
+            name: 'registry.suse.com'
+            location: 'registry.suse.com:5000'
+            insecure: True
+        image: 'registry.suse.de/devel/storage/7.0/containers/ses/7/ceph/ceph'
+
+    There is also the possibility of using overrides in order to set the
+    container registry and image like:
+
+        overrides:
+            ceph_salt:
+                containers:
+                    registry:
+                        name: 'registry.mirror.example.com'
+                        location: 'registry.mirror.example.com:5000'
+                    image: 'quay.io/ceph-ci/ceph'
```

Comment:
This looks like a copy-paste from the cephadm task?

Reply:
yep since there is a plan to move it upstream I just used this as generic example not to have suse only

Reply:
the thing is that `image` value in cephadm is used in a different way, the append :sha1 value to it, not sure if we have the same schema

Reply:
So, I think it can misguide a reader

Reply:
To my pov image is an image, the fact that there can be sha1 or not in the name/path is irrelevant so I don't see anything wrong here.